sdk/avalon_sdk/ethereum/unit_tests/work_order_req.json

```diff
@@ -0,0 +1,28 @@
+{
+    "jsonrpc": "2.0",
+    "method": "WorkOrderSubmit",
+    "id": 29438,
+    "params": {
+        "workOrderId": "76023286e141391c2caac7349235c3d8562de5fb132d81a836032c6f5170a216",
+        "responseTimeoutMSecs": 6000,
+        "payloadFormat": "JSON-RPC",
+        "resultUri": " ",
+        "notifyUri": " ",
+        "workerId": "15f0e2ad6f58f53252bedf7aef1dfeeb767681e57b41548e0a114474f4b87136",
+        "workloadId": "6563686f2d726573756c74",
+        "requesterId": "329e696f600bd707075c470e90e39228edea3a1a681426210eb1df3f6630ee4c",
+        "workerEncryptionKey": "2d2d2d2d2d424547494e20525341205055424c4943204b45592d2d2d2d2d0a4d49494243674b43415145416f70516a6c6b6170674b672f56616b3054723874756f376d67436956662f464c6f6f466d717447716861684a4e392f6571636a4f0a44367353724b42767847473071367078764d6b356f50684439715a3757326d683432534736556f674f6841647066434b5643356b486f6e525a5030696c302b790a72717a72635a4b42647253797a4f53683730563163394679726641445049577431594f746171766f712b6a6a6b48705058414f784336356e38622f73347938310a646c43566a7553457a384c326349397779544e2f3470507433706e774c777a65674177546e327a2f4462496c6c664236564e50394d382b636a6634506d2f6f6a0a467a4f68525578644d6463646451586d7a426372664b55334f74314d35754b4b4b4c485458435250686f3266307a684634336435563430666f792b764d4f556e0a6b6a363346744a7355343873393058547070485435424e645044705a305076524b774944415141420a2d2d2d2d2d454e4420525341205055424c4943204b45592d2d2d2d2d0a",
+        "dataEncryptionAlgorithm": "AES-GCM-256",
+        "encryptedSessionKey": "125C71CE7B19457C6243CCDF440462D0DCFF5BD5370BC0ADFF82260F6EAF04137700A33CB616290167AA99A306430E14EC227806AE7272EA176C7B6CE10A7E4CB3A4847C19B7EC4F6468F5509B29179885EAF96BDA32CF62CE0075EBDE4DF465D97A63C3522FBAE39C474ECF72DA27BFCFE96AEC21F0B2A16FE894CC5B4010F13F50E8AB3999F7614A9F7276D0B9D85F9E3E557EE3039BF4AE91797BF7A90F22E27B01C1B8F4D46902C262BDF07109DCCBA4DE79EBF37BFE5160EE7BD46E7366DB16C8EB7F8084C92765DE01359318D1B443310395106B1517E2F6F1B575DA9DDDA8E9E3A0CF140D09D18412A8BC890892B127F67C0C3EA01CE6DCAAF06DC832",
+        "sessionKeyIv": "C6BECD98005B09850BD31CAB",
+        "requesterNonce": "e5450d21f18b62cacb2cb620728858c7",
+        "encryptedRequestHash": "5934CB58F73305FC7432FB5AA5753A547FAC79BDA2330DF6D1BD466624A7D4D2329FEBD1AB533AE6DB78A7F767DA493F",
+        "requesterSignature": "",
+        "inData": [
+            {
+                "index": 0,
+                "data": "VTq48Uu3gy+Zlge3mUeIRegNfF5CCw=="
+            }
+        ]
+    }
+}
```

Comment:
Add a new line

Reply:
Updated @Ram-srini . Could you please take a look again and merge if things are fine.